app/code/Magento/Customer/Model/Vat.php

```diff
@@ -179,18 +179,21 @@ public function checkVatNumber($countryCode, $vatNumber, $requesterCountryCode =
             return $gatewayResponse;
         }
 
+        $countryCodeForVatNumber = $this->getCountryCodeForVatNumber($countryCode);
+        $requesterCountryCodeForVatNumber = $this->getCountryCodeForVatNumber($requesterCountryCode);
+
```

Comment:
Since Codacity blames about variable name length, you can try renaming them into `$ctryVatCode` and `$requesterCtryVatCode`

Reply:
Hi @aleron75, thanks for the review!

I don't agree with this check on length of variable names by Codacy, I like to keep them readable and understandable, we aren't living in the 1990's anymore where disk space was limited and code was supposed to be as small as possible :)

If Magento code standards really wants me to change these to a shorter name, please let me know, and I will change them, but I've send in PR's in the past where the same check failed, and then it wasn't a problem and was just approved as is.

Reply:
Hello @hostep I will check, if this is not a real issue, your PR will proceed.

Reply:
Hello @hostep, I confirm you can ignore Codacity blames about variable name length.

Reply:
Excellent, thanks! :)